rest-api/src/main/java/gov/cms/qpp/conversion/api/controllers/v1/QrdaControllerV1.java

```diff
@@ -85,13 +85,11 @@ public QrdaControllerV1(final QrdaService qrdaService, final ValidationService v
 	 * @param file the attachment
 	 * @return a supplier that wraps the attachment's input stream retrieval
 	 */
-	Supplier<InputStream> inputStreamSupplier(MultipartFile file) {
-		return () -> {
-			try {
-				return file.getInputStream();
-			} catch (IOException ex) {
-				throw new UncheckedIOException(ex);
-			}
-		};
+	InputStream inputStreamSupplier(MultipartFile file) {
```

Comment:
Since this no longer returns a `Supplier`, please rename the method and JavaDocs.

Reply:
Done